code/datums/topic/admin.dm

```diff
@@ -1051,14 +1051,14 @@
 		return
 
 	if(BSACooldown)
-		to_chat(source.owner, "Standby!  Reload cycle in progress!  Gunnary crews ready in five seconds!")
+		to_chat(source.owner, "Standby! Reload cycle in progress! Gunnery crew ready in five seconds!")
 		return
 
 	BSACooldown = TRUE
 	spawn(50)
 		BSACooldown = FALSE
 
-	to_chat(M, "You've been hit by bluespace artillery!")
+	to_chat(M, "You've been hit by Bluespace Artillery!")
```

Comment:
neither bluespace nor artillery are proper nouns, the original non-capitalization was fine

```suggestion
	to_chat(M, "You've been hit by bluespace artillery!")
```